packages/curve-ui-kit/src/widgets/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import { styled } from '@mui/material/styles'
+import Grid from '@mui/material/Grid2'
+
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+import { LlamaImg } from 'ui/src/images'
+
+import { Description } from './Description'
+import { Section } from './Section'
+import { sections } from './Sections'
+
+const LlamaImageSrc = (LlamaImg as unknown as { src: string }).src
+const Llama = styled('img')({
+  alt: 'Llama',
+  position: 'absolute',
+})
+
+export const Footer = () => (
+  <Grid
+    container
+    component="footer"
+    data-testid="footer"
+    spacing={1}
+    rowGap={4}
+    sx={(t) => ({
+      position: 'relative',
+      paddingInline: SizesAndSpaces.Spacing.lg,
+      paddingBlock: SizesAndSpaces.Spacing.xl,
+      backgroundColor: t.design.Layer[3].Fill,
+    })}
+  >
+    <Grid
+      size={{
+        mobile: 12,
+        tablet: 12,
```

Comment:
No need to repeat if it's the same.. This prop `mobile: 12` can be read as `>mobile` ([mobile breakpoint is 0](https://github.com/curvefi/curve-frontend/blob/88281be8d15553b2718323305d3c3445a80f6656/packages/curve-ui-kit/src/themes/basic-theme/basic.ts#L9))

```suggestion
```

Reply:
It's been removed